mavutil.py

```diff
@@ -1429,7 +1429,11 @@ def _rewind(self):
     def rewind(self):
         '''rewind to start of log'''
         self._rewind()
-        
+
+    def close(self):
+        super().close()
```

Comment:
This won't work on Python2 - and yes, sadly, we're going to continue to support it for a while yet.

```suggestion
        super(mavmmaplog, self).close()
```

Reply:
Sure thing --  I haven't used python2 personally in a while!

Reply:
Lucky you :-)